src/coreclr/inc/gcinfodecoder.h

```diff
@@ -179,7 +179,7 @@ enum ICodeManagerFlags
     ExecutionAborted  =  0x0002, // execution of this function has been aborted
                                  // (i.e. it will not continue execution at the
                                  // current location)
-    AbortingCall      =  0x0004, // The current call will never return
+    // unused         =  0x0004, // was AbortingCall
```

Comment:
```suggestion
```
This information is not important going forward.

Reply:
(You can shrink these bits to be consecutive if you would like - there are multiple unused holes.)

Reply:
I just saw the same pattern decorating gaps in an enum later in the file.

Reply:
The place later in this file where this pattern is used has to do with the GCInfo format. Eliminating those gaps would be GCInfo format breaking change and it is useful to know what that gap was used for in the past versions of the format.

ICodeManagerFlags are not coupled with the file format, so these reasons do not apply here.